payments/guest-wallet/src/main/java/cm/aptoide/pt/guest_wallet/repository/wallet/exception/InvalidWalletException.kt

```diff
@@ -0,0 +1,3 @@
+package cm.aptoide.pt.guest_wallet.repository.wallet.exception
+
+class InvalidWalletException(message: String) : Throwable(message)
```

Comment:
I don't think we need a specific exception here and handle it in a specific way.
This should be a fail fast point for the case that should never happen normally...